### PR TITLE
[IMP] stock: Add new setting for inventory.

### PR DESCRIPTION
**Pull request by mag-odoo**
Description of the issue/feature this PR addresses:
Task:https://www.odoo.com/web#id=29500&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad:https://pad.odoo.com/p/r.70ba2d63e880c3114e4c6e5daa7e839e

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_stock/models/stock_config_settings.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class StockConfigSettings(models.TransientModel):
+    _inherit = 'stock.config.settings'
+
+    security_lead = fields.Float(related='company_id.security_lead', default=lambda self: self.env.user.company_id.security_lead, string="Security Lead Time for Sales")
```
Comment (atp-odoo) [inline addons/sale_stock/models/stock_config_settings.py]: A related is just some specific computed field and should therefore not have a default value.

File: addons/web/static/src/less/views.less [hunk 2]
```diff
@@ -87,3 +87,43 @@
...
+            button.btn-link {
+                padding: 0;
+            }
+        }
+    }
+}
+
+.oe_form_configuration h2 {
+    margin-top: 32px !important;
+}
```
Comment (mdi-odoo) [inline addons/web/static/src/less/views.less]: Move the change to **form_view.less** .

File: addons/web/static/src/js/views/form_upgrade_widgets.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ var AbstractFieldUpgrade = {
         this._super.apply(this, arguments);
         this.get_enterprise_label().after($("<span>", {
             text: "Enterprise",
-            'class': "label label-primary oe_inline"
+            'class': "label label-primary oe_inline oe_enterprise_label"
```
Comment (mdi-odoo) [inline addons/web/static/src/js/views/form_upgrade_widgets.js]: Remove this change .

File: addons/sale_stock/views/stock_config_settings_views.xml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+                            <div class="content-group">
+                                <div class="mt16">
+                                    <field name="module_procurement_jit" class="oe_light_label" widget="radio"/>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="col-md-6 o_setting_box">
+                    <div class="input-group">
```
Comment (mdi-odoo) [inline addons/sale_stock/views/stock_config_settings_views.xml]: Same here .

File: addons/stock_landed_costs/views/stock_config_settings_views.xml [hunk 5]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_stock_config_settings_inherit_landed_cost" model="ir.ui.view">
+        <field name="name">stock.config.settings.inherit.landed.cost</field>
+        <field name="model">stock.config.settings</field>
+        <field name="inherit_id" ref="stock.view_stock_config_settings"/>
+        <field name="arch" type="xml">
+            <div name="landed_cost_info" position="replace">
+                <button name="%(stock_landed_costs.stock_landed_cost_type_action)d" icon="fa-arrow-right" type="action" string="Landed Cost Types" class="oe_link" style="padding-left: 0;"/>
```
Comment (mdi-odoo) [inline addons/stock_landed_costs/views/stock_config_settings_views.xml]: Replace class="oe_link" with class="btn-link" and remove style="padding-left: 0;" .

File: addons/stock_account/models/stock_config_settings.py [hunk 6]
```diff
@@ -7,22 +7,33 @@
...
-    def set_property_valuation_on_product_category(self):
-        data = {0: 'manual_periodic', 1: 'real_time'}
-        return self.env['ir.values'].set_default('product.category', 'property_valuation', data[self.group_stock_inventory_valuation])
-
-    @api.onchange('module_stock_landed_costs')
-    def onchange_landed_costs(self):
-        if self.module_stock_landed_costs:
-            self.group_stock_inventory_valuation = 1
+    def set_default_costing_method(self):
+        return self.env['ir.values'].set_default('product.category', 'new_cost_method', True if self.costing_method else False)
```
Comment (mdi-odoo) [inline addons/stock_account/models/stock_config_settings.py]: Add a new line at the end of file .

**Pull request closed without merging**